scripts/download_file.py

```diff
@@ -213,12 +221,12 @@ def compare_md5(remote_md5: str, local_md5: str) -> str:
             remote_url = args.remote_dir + "/" + fname
 
             print(f"Downloading {remote_url}")
-            if not download_file(remote_url, temp_local_file):
+            if download_file(remote_url, temp_local_file) != True:
                 print(f"Unable to download {remote_url} to {temp_local_file}")
                 sys.exit(1)
```

Comment:
This chunk of code could be replaced by an assert statement. (sorry, just noticed this because I recently did a deep dive on various applications of `assert`)

```python
# download_file() returns a boolean. If the bool, is False, kill the script and 
# raise an AssertionError with the specified message
assert download_file(remote_url, temp_local_file), 
    f"Unable to download {remote_url} to {temp_local_file}"
```